### PR TITLE
Fix CA issue in docker

### PR DESCRIPTION
**Pull request by bikochan**
The docker image was missing ca-certificates which caused a self-signed cert error

### CODE REVIEW

**Comment by unguiculus** [comment]
Thanks for the PR. Pleae fix DCO.

File: Dockerfile [hunk 1]
```diff
@@ -1,5 +1,8 @@
 FROM alpine:3.9
 
+RUN apk update && \
+    apk add ca-certificates
```
Comment (unguiculus) [inline Dockerfile]: Use this instead, which avoids caching the index and performs an update implicitly:

`RUN apk --no-cache add ca-certificates`

Comment (bikochan) [reply Dockerfile]: I'm almost tempted to do a FROM scratch but you probably use alpine for troubleshooting so I'm not changing it.

**Comment by desaintmartin** [comment]
@unguiculus Could you release a new version containing this fix (and other since 0.2.2)?

**Pull request merged**